quiz/migrations/0001_initial.py

```diff
@@ -41,17 +41,17 @@ class Migration(migrations.Migration):
                 ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                 ('text', models.CharField(max_length=300)),
                 ('score', models.IntegerField()),
-                ('quiz', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to='quizz.Quiz')),
```

Comment:
(note for future) Be careful with modifying the migration files. The sequence of migration files is like the commits from git. They are linked in a chain, so if you rewrite history upstream, the chain from other people will not match so the 'pull' (in git case) or 'migrate' (in django migrations) will fail or the changes will not be applicable on the others DBs.

This time it works because it creates entirely new tables (since the app name has changed). This is also a very sensitive point (app name change) because all tables will be duplicated but empty.